ui/components/ui/chip/chip.js

```diff
@@ -106,4 +115,12 @@ Chip.propTypes = {
    * max-content can overflow the parent's width and break designs
    */
   maxContent: PropTypes.bool,
+  /**
+   * Icon location
+   */
+  leftIconUrl: PropTypes.string,
```

Comment:
Should we default `leftIconUrl` to `''`?

Reply:
Fixed in: dc67e851ff3027e78e329448e64b8f8310629bf8